target/linux/ath79/image/generic-ubnt.mk

```diff
@@ -117,3 +117,33 @@ define Device/ubnt_unifiac-pro
   SUPPORTED_DEVICES += ubnt-unifiac-pro
 endef
 TARGET_DEVICES += ubnt_unifiac-pro
+
+define Device/ubnt_routerstation
+  DEVICE_PACKAGES := kmod-usb-ohci kmod-usb2 fconfig
+  ATH_SOC := ar7161
+  IMAGE_SIZE := 16128k
+  IMAGES := sysupgrade.bin factory.bin
+  IMAGE/factory.bin := append-rootfs | pad-rootfs | mkubntimage | check-size $$$$(IMAGE_SIZE)
+  IMAGE/sysupgrade.bin := append-rootfs | pad-rootfs | combined-image | check-size $$$$(IMAGE_SIZE) | append-metadata
+  KERNEL := kernel-bin | append-dtb | lzma | pad-to $$(BLOCKSIZE)
+  KERNEL_INITRAMFS := kernel-bin | append-dtb | pad-to $$(BLOCKSIZE)
```

Comment:
I'm quite sure the padding isn't required for the initramfs image

Reply:
k